reference.md

```diff
@@ -32,8 +32,10 @@ Here are some helpful terms that we use at 2i2c.
 
 ```{glossary}
 Single Source of Truth
+SST
   Remote teams and open communities need to balance information across team members, and ensure that everyone is on the same page. For this reason, it is recommended to adopt a "single source of truth" for anything important [^sst1]. This is an authoritative source that everyone can look to in order to know the current status and plan for anything we do at 2i2c.
 
+International Computer Science Institute
 ICSI
```

Comment:
Is either this or the other ordered in the wrong way? I'm not sure how the glossary directive works but perhaps an abbreviation is supposed to come first or last?

Reply:
I think either one of them become terms that you can include with {term}, the order shouldn't matter

Reply:
Ah makes sense!